torchdynamo/debug_utils.py

```diff
@@ -19,6 +19,33 @@ def minifier_dir():
     return f"/tmp/minifier_{getpass.getuser()}"
 
 
+@functools.lru_cache(None)  # subprocess is expensive
+def _cuda_system_info_comment():
```

Comment:
Meta comment - if you pull this into its own PR, I can stamp this. I do not know the rest of the code well enough to do so.

Reply:
Yeah, I snuck this one in because it seemed like a tiny change.  I was noticing this function take half of some profiles.  :)